images/Dockerfile

```diff
@@ -16,350 +16,499 @@
 #
 # ***************************************************************
 
-ARG BASE_YUM_REPO=release
-ARG BASE_OSG_SERIES=23
+# For selecting OSG-maintained packages.
 ARG BASE_OS=el9
+ARG OSG_SERIES=23
+ARG OSG_REPO=release
+
+# For compiling Pelican.
+ARG GO_VER=1.21.6
+ARG NODEJS_VER=20
+
+# For compiling LotMan.
+ARG JSON_VERSION="v3.11.3"
+ARG JSON_SCHEMA_VALIDATOR_VERSION="2.3.0"
+ARG LOTMAN_VERSION="v0.0.4"
```

Comment:
Similarly, looks like we have lotman available in Koji as well: https://koji.osg-htc.org/koji/buildinfo?buildID=17613.  Another thing we could skip building.

It's less clear about `json-schema-validator`; there's a version in there but it's somewhat older.

Regardless, let's pin the build version where we can.